content-services/latest/upgrade/index.md

```diff
@@ -246,3 +252,23 @@ The `dir.root` directory is defined in the `alfresco-global.properties` file. By
 2. Point the new deployment to the old database using the `db.*` properties in the `alfresco-global.properties` file by providing the JDBC URL, database name, log in credentials, and any other relevant configuration options.
```

Comment:
Should we add a step before pointing to the old database, about taking an up-to-date backup of the live environment, as the upgrade process may have started days before? We mention the backup in the prerequisites, but this is probably the correct point in the flow to remind them :-)